src/schemas/global.ts

```diff
@@ -22,6 +22,7 @@ export default {
       required: ['id'],
       properties: {
         id: { $ref: '#/definitions/uuid' },
+        templateId: { $ref: '#/definitions/uuid' },
```

Comment:
Not sure this needs to be here. Which template id this the id of, if this specific to the task of a creating the templates ?

Reply:
Remove it from here and update the local schema to include the template_id param